.drone.yml

```diff
@@ -52,45 +52,26 @@ steps:
    - push
    - pull_request
 
-#- name: Unit Tests and SonarQube Scanner
-#  pull: if-not-exists
-#  image: quay.io/ukhomeofficedigital/sonar-scanner-node:9fed2b90853321ac9a9c3da548a180f22c5dc4cb
-#  commands:
-#  - export NOTIFY_API_KEY=dummy
-#  - cd src
-#  - npm install
-#  - npm run test-drone
-#  - cd ..
-#  - sonar-scanner -Dsonar.projectVersion=$${DRONE_BUILD_NUMBER}
-#  settings:
-#    group: build-stage
-#  when:
-#    event:
-#    - push
-#    - pull_request
-
-- name: scan-newly-built-image
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/anchore-submission:latest
+- name: scan-image
   pull: always
-  failure: ignore
```

Comment:
Question: should the `failure: ignore` be removed?
- I am in favour of it being removed, since it means we are likely to fix security incidents as they come.
- Would there be any reason not to e.g., persistent but irrevelant security issues at the image level?

Reply:
good point, I agree, the image build is easy to sort it's a matter of triggering a building and the image updates

Reply:
At image level we should always try to fix as we go as it does not take much time, I will put up docs regarding fixing the images which is easy to do

Reply:
Ok, that is cool, that clarifies.
* I am happy for a docs update, but it is not required.
* Unless there is an issue I am missing, I am happy for it to be merged.